src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -8093,7 +8093,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>Extension declarations may not have a name.</value>
   </data>
   <data name="ERR_ExtensionDisallowsMember" xml:space="preserve">
-    <value>Extension declarations can include only methods or properties</value>
+    <value>Extension declarations can include only methods, non-conversion operators or properties</value>
```

Comment:
Is the plan to potentially support conversion operators later or is there a reason they cannot be supported?

Reply:
> Is the plan to potentially support conversion operators later or is there a reason they cannot be supported?

At the moment there is no plan to support conversions, see https://github.com/dotnet/csharplang/blob/main/meetings/working-groups/extensions/extension-operators.md.

> Note: Does not cover user-defined implicit and explicit conversion operators, which are not yet designed or planned.